docs/current/quickstart/snippets/test/index.mjs

```diff
@@ -0,0 +1,23 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  // use a node:16-slim container
+  // mount the source code directory on the host
+  // at /src in the container
+  const source = await client.container()
+    .from("node:16-slim")
+    .withMountedDirectory('/src', client.host().directory('.', { exclude: ["node_modules/", "ci/"] }))
```

Comment:
see :
https://github.com/dagger/dagger/pull/4391/files#r1087936349 and https://github.com/dagger/dagger/pull/4391/files#r1087939838 regarding directory path